libusb/core.c

```diff
@@ -902,6 +903,17 @@ void API_EXPORTED libusb_free_device_list(libusb_device **list,
 	free(list);
 }
 
+/** \ingroup libusb_dev
+ * Get the backend-specific handle (session data in terms of libusb) of a device.
+ * Can be used for direct low-level access to the backend when necessary.
+ * \param dev a device
+ * \returns the backend-specific handle (session data)
```

Comment:
> lets avoid the term `handle` - it might be confusing

Good point. It's not a handle, indeed. I tried to rewrite the comment and make it more clear. I'm not a native English speaker, though. It could make sense for a native speaker to review/edit this comment (it goes to docs if merged).

Reply:
> I'm not a native English speaker

Neither do I, but the updated version is much better, thanks.